File: MPI/mpicheck/tests/functions/1.0/MPI_Test_cancelled.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    const MPI_Status *var_0;
    int *var_1;
    int ret;
    /* calls */
    ret = MPI_Test_cancelled(var_0, var_1);
    ret = PMPI_Test_cancelled(var_0, var_1);
    return 0;
}

File: MPI/mpicheck/tests/functions/1.0/MPI_Test_cancelled.f90

        
        program main
        use mpi
        
        INTEGER var_0(10)
       LOGICAL var_1
       INTEGER var_2
        call mpi_test_cancelled(var_0, var_1, var_2)
       call pmpi_test_cancelled(var_0, var_1, var_2)
        end program main
    